bluesky/plan_stubs.py

```diff
@@ -650,6 +650,48 @@ def unsubscribe(token):
     return (yield Msg('unsubscribe', token=token))
 
 
+def install_suspender(suspender):
+    """
+    Install a suspender during a plan.
+
+    Parameters
+    ----------
+    suspender : :class:`bluesky.suspenders.SuspenderBase`
+        The suspender to install
+
+    Yields
+    ------
+    msg : Msg
+        Msg('install_suspender', suspender)
```

Comment:
Can this be changed to `Msg('install_suspender', None, suspender)`? As strange as that signature is -- and I'm wincing as I make this suggestion! -- `Msg` returns a `namedtuple`, and we have thus far reserved the `obj` attribute to always be a Device/Signal if present. See for example `Msg('sleep', None, 5)` and `Msg('subscribe', None, callback)`. That signature is, to state it charitably, _surprising_, which is part of the reason we have plan_stubs, to layer over an API customized to the situation.

Reply:
Ahh, that explains a lot! I'll adjust accordingly.